src/Core/composer.json

```
@@ -7,8 +7,9 @@
     "Apache-2.0"
   ],
   "require": {
-    "php": "~7.0.13|~7.1.0|~7.2.0",
+    "php": "~7.1.3||~7.2.0",
```

@MicheleBeargroup , @lemked observed this and I was curious if this was intentional?

---

Yeah, it's a requirement in the BUNDLE-1383.

---

Why the double pipe (`||`) ? Should it instead be a single pipe?
```suggestion
    "php": "~7.1.3|~7.2.0",
```

---

@MicheleBeargroup ^^